ingestion/create_cadet_databases_source/source.py

```diff
@@ -46,6 +48,14 @@
 }
 
 
+subject_areas_filepath = os.path.join(
```

Comment:
Might be worth extracting this to a function in utils.py so we can use it in other places